packages/keystone/fields/RegexplessText/index.js

```diff
@@ -0,0 +1,25 @@
+const { Text } = require('@keystonejs/fields')
+
+class RegexplessKnexFieldAdapter extends Text.adapters.knex {
+
+    equalityConditionsInsensitive (dbPath) {
+        return {
+            [`${this.path}_i`]: value => b => b.whereRaw(`lower(${dbPath}) = lower(?)`, [value]),
+            [`${this.path}_not_i`]: value => b =>
+                value === null
+                    ? b.whereNotNull(dbPath)
+                    : b.whereRaw(`lower(${dbPath}) != lower(?)`, [value]).orWhereNull(dbPath),
+        }
+    }
+}
+
+module.exports = {
+    type: 'RegexplessText',
```

Comment:
Maybe just `Text`?

Reply:
`Text` associates with the build-in type. I'd like another name. Anyway, we may change this name if needed.

Reply:
It’s really better to use Text. Some checks expect this type name.

Reply:
Ok :-/